namespaces/live.cloud-platform.service.justice.gov.uk/data-platform-app-officehub-cms-app-dev/01-rbac.yaml

```diff
@@ -0,0 +1,16 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: data-platform-app-officehub-cms-app-dev-admin
+  namespace: data-platform-app-officehub-cms-app-dev
+subjects:
+  - kind: Group
+    name: "github:officehub-cms-app-v2-admin-team"
```

Comment:
This needs to be the team that's actually the admin of that repo. When we did migrations, we created the teams which is why they have such weird name but in your case, one already exists, so it should be added here.
In your case it should be `github:ltd-pa`

Reply:
Thanks - now resolved :)